vision/objectdetection/color_detector.go

```diff
@@ -24,14 +26,23 @@ func NewColorDetector(cfg *ColorDetectorConfig) (Detector, error) {
 		return nil, err
 	}
 	hue, _, _ := col.HsvNormal()
-	tol := cfg.Tolerance
+	tol := cfg.HueTolerance
 	if tol > 1.0 || tol < 0.0 {
 		return nil, errors.Errorf("tolerance must be between 0.0 and 1.0. Got %.5f", tol)
 	}
 
+	sat := cfg.SaturationCutoff
+	if sat <= 0.0 {
```

Comment:
Do we think this kinda "check 0 and set default" thing is valid for HueTolerance also? Or no since it's not really an optional input? In other words... do we want the user to be able to set hueTolerance = 0?

Reply:
Per adding data-500 into this, I'm using 0=default and using 0.05 as the default.